userguide/storage.rst

```diff
@@ -189,11 +189,10 @@ and save the key to a safe location. When finished, click
 :guilabel:`DONE`.
 
 The screen refreshes after the pool is created, showing the new pool
```

Comment:
Didn't we remove this sentence entirely in one version?  It does not really add any information helpful to the user.

Reply:
I checked all open PRs and merged angulargui back into this branch. There are no changes to this sentence.